Writerside/topics/analysis/domain/raptor.md

```diff
@@ -29,49 +29,34 @@ graphs that incorporate the complexity of transfers. This characteristic makes R
 networks of any size, as it avoids the need for time-consuming preprocessing. This efficiency allows us to enable
 features like accessibility and bike information on our routes.
 
-## Sequence Diagram RAPTOR (simple)
+## Sequence Diagram RAPTOR (simplified)
```

Comment:
since we don't have a more detailed sequence diagram, I would rename to:
## Conceptual Sequence Diagram RAPTOR